GSG final steps - Style edits and move to top level (#1073)
timescale/docs: getting-started/create-cagg/create-cagg-policy.md

<procedure>

### Manually refreshing a continuous aggregate

1.  Refresh the continuous aggregate `stock_candlestick_daily` for data within
    the past week:

    ```sql

>     CALL refresh_continuous_aggregate('stock_candlestick_daily',now() - interval '1 week', now());

Reviewer: ```suggestion
    CALL refresh_continuous_aggregate(
      'stock_candlestick_daily',
      now() - INTERVAL '1 week',
      now()
    );
```

    ```

1.  This manual refresh updates your continuous aggregate only once. It doesn't
    keep the aggregate up to date automatically. To set up an automatic refresh
    policy, see the preceding section on continuous aggregate refresh policies.

</procedure>

## Next steps

> Now that you have a continuous aggregates setup and refreshing automatically,

Reviewer: ```suggestion
Now that you have a continuous aggregate set up and refreshing automatically,
```

For detailed information on continuous aggregates and real-time aggregation,
see the [continuous aggregates section][continuous-aggregates].

[getting-started-compression]:/getting-started/compress-data/
[flightaware]: https://blog.timescale.com/blog/how-flightaware-fuels-flight-prediction-models-with-timescaledb-and-grafana/
[crypto-bot]: https://blog.timescale.com/blog/how-i-power-a-successful-crypto-trading-bot-with-timescaledb/

> [continuous-aggregates]: /how-to-guides/continuous-aggregates

Reviewer: ```suggestion
[continuous-aggregates]: /timescaledb/:currentVersion:/how-to-guides/continuous-aggregates
```
